cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -381,6 +385,21 @@ mkTracers traceConf tracer bcCounters = do
                   _ -> pure ()
               _ -> pure ()
 
+    notifyForkIsCreated :: Tracer IO (WithSeverity (ChainDB.TraceEvent blk))
+    notifyForkIsCreated =
+        Tracer $ \(WithSeverity _ ev') ->
+          case ev' of
+              (ChainDB.TraceAddBlockEvent ev) -> case ev of
+                  ChainDB.SwitchedToAFork _ _ _ -> do
```

Comment:
If you use `ChainDB.SwitchedToAFork {}` there will no error message when the number of fields changes.